### ISSUE TITLE
Logging does not work

### ISSUE THREAD

**Issue by MarcusRoeckrath** [ola-ct/Qt-SESAM#124]
Starting Qt-SESAM in a console it is reported

Logger writing to /home/marcus/.local/share/ct/QtSESAM/QtSESAM.log

I coud not find a log file in this location nor in any other location of my home dir.

The directory /home/marcus/.local/share/ct does not exists. Creating this directory manually does not help to get logging to work.

**Comment by MarcusRoeckrath** [comment]
Now I got it work:

I activated the backup feature which created the missing directory /home/marcus/.local/share/ct/QtSESAM.

Logging does only work if this directory exists but did not create it if missing.

I made a new test without backup functionality an deleting the log directory.

Logging does work but when I created /home/marcus/.local/share/ct/QtSESAM manually then logging worked again.

The log routine should create /home/marcus/.local/share/ct/QtSESAM if this directory is missing.
